modules/Installer/AWS.Tools.Installer.psm1

```diff
@@ -488,7 +453,7 @@ function Install-AWSToolsModule {
         $RequiredVersion = Get-CleanVersion $RequiredVersion
         $MaximumVersion = Get-CleanVersion $MaximumVersion
 
-        Write-Verbose "[$($MyInvocation.MyCommand)] ConfirmPreference=$ConfirmPreference WhatIfPreference=$WhatIfPreference VerbosePreference=$VerbosePreference Force=$Force"
+        Write-Verbose "[$($MyInvocation.MyCommand)] ConfirmPreference=$ConfirmPreference WhatIfPreference=$WhatIfPreference VerbosePreference=$VerbosePreference Force=$Force Name=($Name) RequiredVersion=$RequiredVersion SkipUpdate=$SkipUpdate CleanUp=$CleanUp"
```

Comment:
I will take some time and think about this. I like the suggestion and it would be better to have all parameters logged. I would prefer having them on a single line so that we don't spam to much text for functions that are being called a lot like `Get-AWSToolsModuleDependenciesAndValidate`.